src/dxtbx/format/FormatNXmxEigerFilewriter.py

```diff
@@ -0,0 +1,90 @@
+from __future__ import annotations
+
+import re
+
+import h5py
+
+from scitbx.array_family import flex
+
+import dxtbx.nexus
+from dxtbx.format.FormatNXmx import FormatNXmx
+from dxtbx.nexus import _dataset_as_flex, get_detector_module_slices
+
+DATA_NNNNNN_RE = re.compile(r"data_\d{6}")
```

Comment:
I know what NNNNNN is about here but NNNNNN looks prone to later typos? `DATA_FILE_RE` seens safer to me